.rubocop.yml

```diff
@@ -20,9 +24,27 @@ Metrics/BlockLength:
     - 'spec/**/*.rb'
     - 'openfeature-sdk.gemspec'
 
+Gemspec/DevelopmentDependencies:
+  EnforcedStyle: gemspec
+
 Gemspec/RequireMFA:
   Enabled: false
 
 Style/DocumentDynamicEvalDefinition:
   # TODO re-enable after figuring out what it actually wants
   Enabled: false
+
+RSpec/ContextWording:
+  Enabled: false
```

Comment:
Since we're using the OpenFeature specification language, disabling this rule as it will constantly be violated.